### PR TITLE
Fixes #15713: Show log informations next to reports and full compliance report

### PR DESCRIPTION
**Pull request by VinceMacBuche**
https://issues.rudder.io/issues/15713

Replacing previous PR: https://github.com/Normation/rudder/pull/2493

### CODE REVIEW

**Comment by VinceMacBuche** [comment]
PR rebased

**Comment by VinceMacBuche** [comment]
PR rebased

**Comment by VinceMacBuche** [comment]
PR rebased

**Comment by VinceMacBuche** [comment]
Commit modified

**Comment by VinceMacBuche** [comment]
Commit modified

**Comment by VinceMacBuche** [comment]
Commit modified

File: webapp/sources/rudder/rudder-web/src/main/scala/com/normation/rudder/web/services/LogDisplayer.scala [hunk 1]
```diff
@@ -217,6 +219,7 @@ case class ReportLine (
 
       JsObj(
           ( "executionDate", executionDate.toString("yyyy-MM-dd HH:mm:ss") )
+        , ( "runDate", runDate.toString("yyyy-MM-dd HH:mm:ss") )
```
Comment (fanf) [inline webapp/sources/rudder/rudder-web/src/main/scala/com/normation/rudder/web/services/LogDisplayer.scala]: Can you use either ISODateTimeFormater .dateTime or our date formatter service with "getDisplayDate" ? We need at least timezone and it's more consistant that way

**Comment by VinceMacBuche** [comment]
PR rebased

**Comment by VinceMacBuche** [comment]
Commit modified

**Comment by fanf** [comment]
When we first come to the page, all event logs are loaded, which leads to a very bad experience (huge latency...). 
By default, there must be a "oldest date" setted to yesterday. 
It may be a problem linked to: https://issues.rudder.io/issues/15945
If so, in case when dates are not parsable, we should get an error, not "load everything" which is very bad for rudder load.

**Comment by fanf** [comment]
bad PR, sorry !

**Pull request merged**